Realm.podspec

```diff
@@ -33,22 +33,31 @@ Pod::Spec.new do |s|
                               'include/Realm/RLMResults.h',
                               'include/Realm/RLMSchema.h',
                               'include/Realm/Realm.h',
+
+                              # Realm.Dynamic module
                               'include/Realm/RLMRealm_Dynamic.h',
                               'include/Realm/RLMObjectBase_Dynamic.h'
 
-  private_header_files      = 'include/Realm/*{Accessor,RealmUtil,ListBase,ObjectStore,Private,SwiftSupport}.h',
-                              'include/realm.hpp',
-                              'include/realm/realm_nmmintrin.h',
-                              'include/realm/util/*.h',
-                              'include/realm/**/*.hpp',
-                              'include/Realm/**/*.hpp'
+                              # Realm.Private module
+  private_header_files      = 'include/Realm/RLMAccessor.h',
```

Comment:
all the hardcoded paths are unfortunate here. Can't we glob `include/Realm/*_Private.h` and then explicitly include the 4 others?